store/generated_key/memstore/memconfig/config.go

```diff
@@ -17,6 +17,8 @@ type Config struct {
 	// after sleeping PutLatency duration.
 	// This can be used to simulate eigenda being down.
 	PutReturnsFailoverError bool
+	// if true, memstore is enabled and will be used
+	Enabled bool
```

Comment:
Why are you adding this here? I think we had this outside right?
I originally thought about having it here, but think it can lead to some conflicts. Like what if we add another backend, which also has an enabled flag? I feel like what we want is the external config to be something like:
```
backend: enum(memstore, eigenda, ...)
memstoreconfig{..}
eigendaconfig{..}
otherbackendconfig{..}
```

So feels more natural to me if the backend type is outside of any single one backend config?

Reply:
[fe4b76c](https://github.com/Layr-Labs/eigenda-proxy/pull/344/commits/fe4b76c3d5ac5ae938eb886dbc8b7daf770c32cb)